homeassistant/components/vicare/config_flow.py

```diff
@@ -39,6 +46,11 @@
 USER_SCHEMA = REAUTH_SCHEMA.extend(
     {
         vol.Required(CONF_USERNAME): cv.string,
+    }
+)
+
+OPTIONS_SCHEMA = vol.Schema(
+    {
```

Comment:
Any reason to not use a `SelectSelector` so we can benefit from translating the options?

Reply:
I could not figure out how to do the translation. I added `SelectSelector` with a plain list for now.

Reply:
Set a translation key and then add `selector` with it's options to `strings.json`.
Example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/scrape/strings.json#L139